src/utils.js

```diff
@@ -43,3 +43,5 @@ export function safeVariableName(name) {
 	const identifier = normalized.replace(INVALID_ES3_IDENT, '');
 	return camelCase(identifier);
 }
+
+export const EXTENSION = /(\.(umd|cjs|es|m))?\.([cm]?[tj]sx?)$/;
```

Comment:
Names are hard... 😅

Certainly open to suggestions

Reply:
Also (horrifyingly) supports the [`.cts` and `.mts` extensions](https://www.typescriptlang.org/docs/handbook/esm-node.html), as it looks like TS might head down that path